mite_browser/__init__.py

```diff
@@ -41,11 +41,11 @@ def url_builder(base_url, *args, **kwargs):
     return url
 
 
-def browser_decorator(separation=0):
+def browser_decorator(separation=0, embedded_res=False):
     def wrapper_factory(func):
         @mite_http
```

Comment:
While you are making changes, do you mind adding `wraps(func)` here (from functools)?  It helps with debugging in a few misc ways.  I've recently added it to other parts of mite (https://github.com/sky-uk/mite/pull/190) but I didn't do mite_browser because I (mistakenly, it turns out) thought it was in legacy-only mode.

Reply:
Done